llmfoundry/callbacks/eval_gauntlet_callback.py

```diff
@@ -19,6 +19,21 @@ class Weighting(Enum):
     LOG_SAMPLE_SZ = 3
 
 
+class ModelGauntlet(Callback):
```

Comment:
Ah I didn't notice initially that the file name changed too. So, this should actually be in the original file name `model_gauntlet_callback.py`